console/src/components/Services/Events/EventTriggers/Modify/OperationEditor.tsx

```diff
@@ -50,6 +51,16 @@ const OperationEditor = (props: OperationEditorProps) => {
     setOperationColumns(existingOpColumns);
   };
 
+  const editorToggle = (opCols: ETOperationColumn[]) => {
```

Comment:
I think the argument `opCols` is not necessary, I think you can use `operationColumns` directly here since it is a prop value that we are dependent upon here.